recipe-server/client/control_new/components/recipes/Details.js

```diff
@@ -0,0 +1,45 @@
+import { Card } from 'antd';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function Details({ recipe }) {
+  return (
+    <div className="recipe-details">
+      <Card key="recipe-details" title="Recipe">
```

Comment:
Oh I see it's in the docs. This was definitely a super-recent addition. We probably need to upgrade ant. But I definitely don't want to do that in this PR.